plugins/mcreator-localization/lang/texts.properties

```diff
@@ -2329,6 +2337,15 @@ dialog.setup_workspace.step.gradle_daemons=Stopping running Gradle daemons
 dialog.setup_workspace.step.gradle_project=Setting up Gradle project
 dialog.setup_workspace.step.importing_gradle=Importing Gradle project
 dialog.setup_workspace.step.generating_base=Generating base source
+dialog.setup_workspace.step.failed_gradle_caches=Failed to reload Gradle caches
+dialog.setup_workspace.step.failed_build_caches=Failed to build cache. Reason: 
+dialog.setup_workspace.step.workspace_setup_fail=<html><big>Workspace setup failed</big><br><br>The setup of MCreator failed. Based on the suggestions you got, try to fix any errors you have and run it again. <br><br><b>Check https://mcreator.net/support/knowledgebase for tips on fixing the setup errors.</b><br><br>If this is the first time you are seeing this message, try re-running the setup first.<br><br>If the error is fixable by you (internet connection problems, ...), you can fix the problem and click the first button<br><b>after</b> you fix the problem. MCreator will run the setup again automatically. You choose to copy the error log to<br>the clipboard for further investigation. You can close the MCreator and run it again when you think you fixed the problem.
```

Comment:
Split this into more lines at <br>, like this:

![image](https://user-images.githubusercontent.com/16374228/123983147-ee03ab00-d9c3-11eb-96c8-8f77fbf9ed81.png)